src/Components/Legend.jsx

```diff
@@ -0,0 +1,137 @@
+          direction='bottom'
+          visible={this.props.visible}
+          onClick={this.hideLegend}
+        >
+          <Grid >
+            <Grid.Column textAlign='center' className="legend-padding" >
+              <Icon
+                name='window minimize outline' 
+                color='grey' 
+                className="legend-icon" 
+                size='large' 
+              />
+
+              <br></br>
+
+            </Grid.Column>
+          </Grid>
+
+          <Grid className='legend-grid'>
+          <Grid.Column 
+          className='legend-grid-column-level'
+          width={6}>
+              <Header className='level-text-legend' as='h4'>
+                Me at
+              </Header>
+
+              <Header className='level-text-legend' as='h4'>
+                Newbie at
+              </Header>
+
+              <Header className='level-text-legend' as='h4'>
+                Settled at
+              </Header>
+
+            </Grid.Column>
+
+            <Grid.Column 
+            centered
+            width={5}
+            className='legend-grid-column'
+            >
+            <Header as='h4'>
+                <Icon
+                  id='circle-my-work'
+                  name='circle'
+                  size='small'
+                  >
+                </Icon>
+                work
+              </Header>
+
+              <Header as='h4'>
+                <Icon
+                  id='circle-newbie-work'
+                  name='circle'
+                  size='small'
+                  >
+                </Icon>
+                work
+              </Header>
+
+              <Header as='h4'>
+                <Icon
+                  id='circle-settled-work'
+                  name='circle'
+                  size='small'
+                  >
+                </Icon>
+                work
+              </Header>
+            </Grid.Column>
+
+            <Grid.Column 
+            centered
+            className='legend-grid-column'
+            width={5}>
+              <Header as='h4'>
+                <Icon
+                  id='circle-my-play'
+                  name='circle'
+                  >
+                </Icon>
+                play
+              </Header>
+
+              <Header as='h4'>
+                <Icon
+                  id='circle-newbie-play'
+                  name='circle'
+                  size='small'
+                  >
+                </Icon>
+                play
+              </Header>
+
+              <Header as='h4'>
+                <Icon
+                  id='circle-settled-play'
+                  name='circle'
+                  size='small'
+                  >
+                </Icon>
+                play
+              </Header>
+            </Grid.Column>
+
+          </Grid>
+        </Sidebar>
+      </>
+
+    )
+  }
+}
+const mapStateToProps = (state) => {
+  return {
+    visible: state.animation.legendVisible
+  }
+}
```

Comment:
since this component has access to the redux store, do you think we should make a conditional so that non logged in visitors do not see the `my work` and `my play` options?

Reply:
yes done